addons/mail/static/src/services/new_message_service/new_message_service.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+const { Component } = owl;
+
+export const newMessageService = {
+    dependencies: ["command"],
+    start(newEnv, { command }) {
+        command.add("Message a user", () => {
+            const env = Component.env;
+            const messagingMenu = env.messaging && env.messaging.messagingMenu;
```

Comment:
Until we have proper new service dependency for messaging, this should also await on both messaging created and initialized.
Then you can remove the condition on env.messaging.